avocado/plugins/run.py

```diff
@@ -89,6 +91,10 @@ def configure(self, parser):
         parser.add_argument('--ignore-missing-references', choices=('on', 'off'),
                             help="Force the job execution, even if some of "
                             "the test references are not resolved to tests.")
+        parser.add_argument("--dump-variants", default=None,
+                            help="Dump the Variants to a JSON file")
```

Comment:
I don't think we need `--dump-variants` here, it might give false impression that it'd also store the tests associated with the execution and other things. Do you think people will run job just to produce the variants? I think having it in `variants` is enough.

Reply:
Yes, I was unsure about this. Let me remove it and and we can re-consider in case some user asks for it.